src/ripple/basics/impl/CheckLibraryVersions.cpp

```diff
@@ -81,18 +78,32 @@ void checkVersion(std::string name, std::string required, std::string actual)
 
 void checkBoost(std::string version)
```

Comment:
I don't have a strong opinion, but I normally think of read-only params being passed by const ref.